appveyor.yml

```diff
@@ -11,33 +11,40 @@ version: 1.0.{build}
 # Build worker image (VM template)
 image: Visual Studio 2015
 
-# build configuration, i.e. Debug, Release, etc.
-configuration:
-  - Release
-  - PythonModule
+environment:
+  matrix:
+    - JOB: Release_x86
+      configuration: Release
+      platform: x86
+    - JOB: Release_x64
+      configuration: Release
+      platform: x64
+    - JOB: Python2.7
+      PYTHON: "C:\\Python27-x64"
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
```

Comment:
Do we need this change?

Reply:
Am I correct if I say that this is needed for `call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86_amd64` line?

Reply:
Yes. On Appveyor, Python2 otherwise always invoke MSVC++ 9 (Visual Studio 2008 version 9.0). `SET VS90COMNTOOLS=%VS140COMNTOOLS%` just doesn't help here. APPVEYOR_BUILD_WORKER_IMAGE is unfortunately undocumented.